sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -1070,6 +1107,7 @@ public Instant getWatermark() {
 
     @Override
     public CheckpointMark getCheckpointMark() {
+      reportBacklog();
```

Comment:
1. The backlog is most accurate inside `updateLatestOffsets()` which runs every 5 seconds. Rest of the time, we are slightly undercounting the backlog.
 1. The backlog reported to runner happens inside `getSplitbacklogBytes()` which get called for every bundle or so. 

Either of these two places sounds better than `getCheckpointMark()`.  (2) is probably what we should do since it keeps it consistent with what gets reported.